src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -519,20 +540,24 @@ public async Task<bool> StartFinancialReview(Guid applicationId, string reviewer
         {
             using (var connection = new SqlConnection(_config.SqlConnectionString))
             {
-                await connection.ExecuteAsync(@"UPDATE Apply SET FinancialReviewStatus = @financialReviewStatusInProgress,
-                                                UpdatedAt = @updatedAt, UpdatedBy = @updatedBy
-                                                WHERE ApplicationId = @applicationId
-                                                AND apply.ApplicationStatus = @applicationStatusGatewayAssessed AND apply.DeletedAt IS NULL
-                                                AND FinancialReviewStatus IN ( @draftStatus, @newStatus )",
+                await connection.ExecuteAsync(@"if exists(select * from apply where ApplicationId = @applicationId 
+                                                        AND apply.ApplicationStatus = @applicationStatusGatewayAssessed AND apply.DeletedAt IS NULL)
+                                                        BEGIN
+                                                        if not exists (select * from FinancialReview where ApplicationId = @applicationId)
+	                                                        BEGIN
+	                                                         insert into FinancialReview (applicationId, Status)
+		                                                        values (@applicationId, @financialReviewStatusInProgress)
+
+	                                                        update Apply set updatedBy =@updatedBy, updatedAt=@updatedAt where applicationId = @applicationId
+	                                                        END
+                                                        END",
```

Comment:
Looks a bit long winded. Could both exist statements be merged into one with an appropriate join & where clause?

Reply:
as discussed, leaving as is